.circleci/config.yml

```diff
@@ -1,40 +1,98 @@
-version: 2
-jobs:
-  build-and-test:
+version: 2.1
+
+executors:
+  job-environment:
+    shell: /bin/bash --login -eo pipefail
     macos:
-      xcode: "10.2.0"
+      xcode: 11.2.0
+    environment:
+      LC_ALL: en_US.UTF-8
+      LANG: en_US.UTF-8
 
+commands:
+  prepare:
     steps:
-      - checkout
-      - run: | 
+      - run: |
+          echo "ruby-2.6.5" > ~/.ruby-version
           bundle install --without=development
-      - run:
-          name: Bootstrap
-          command: bundle exec fastlane ios bootstrap
-      - run:
-          name: Run test suite
-          command: bundle exec fastlane ios ci
-          environment:
-            SCHEME: SimpleKeychain-iOS
-            DEVICE: iPhone 8
-            FASTLANE_EXPLICIT_OPEN_SIMULATOR: 2
-      - run:
-          name: Pod Lib Lint
-          command: bundle exec fastlane ios pod_lint
-      - run:
-          name: SwiftPM Test
-          command: swift test
+          grep -lR "shouldUseLaunchSchemeArgsEnv" *.* --null | xargs -0 sed -i '' -e 's/shouldUseLaunchSchemeArgsEnv = "YES"/shouldUseLaunchSchemeArgsEnv = "YES" codeCoverageEnabled = "YES"/g'
+      - restore_cache:
+          keys:
+            - carthage-cache-{{ checksum "Cartfile.resolved" }}
```

Comment:
add a default key
```suggestion
            - carthage-cache-{{ checksum "Cartfile.resolved" }}
            - carthage-cache
```

Reply:
But if the file doesn't exist, there won't be a cache folder in the first place

Reply:
And the file is in version control, so it should be there